### ISSUE TITLE
Consent für MAKI-Forschungsprojekt und Datenübermittlung nach Tibber-Integration

### ISSUE THREAD

**Issue by timobaehr** [ehwplus/app-backlog#522]
- Wir informieren alle deutschen Nutzer nach App-Update, dass wir ab jetzt Tibber Pulse unterstützt. Außerdem erscheint eine Info für Nutzer, wo man sich diese Hardware kaufen kann. Wenn ein Nutzer das daraufhin einrichtet, dann fragen wir ihn, ob er am Forschungsprojekt teilnehmen will.
- Nutzer, die bereits den Tibber Pulse eingerichtet haben, fragen wir direkt einmalig, ob sie am Forschungsprojekt teilnehmen möchten.
- Nach Abholen der Daten von der API übermitteln wir alle historischen Stundenverbräuche an die Schnittstelle der Goethe-Universität
- objectId und regionCode (Bundesland) mitübermitteln
- Für welche Stunden Hinweis einblenden

**Comment by 0phelia** [comment]
@timobaehr Gibt's dafür ein Layout in Figma? Habe ein branch angelegt, wo es folgendeweise funktioniert:

https://github.com/user-attachments/assets/7fa75d95-3508-4e26-8eee-bb0003e7e6c0
https://github.com/user-attachments/assets/03f32c4f-c559-4016-892c-54bd8498cc57

Würde mich über ein Feedback freuen :)

**Comment by timobaehr** [comment]
Nein, speziell für MAKi habe ich leider kein Layout erstellt. Aber sieht super aus wie jetzt umgesetzt. Kann genauso bleiben. :)
